recipes/MEDIA/media_prepare.py

```diff
@@ -0,0 +1,910 @@
+            + "/E0024/MEDIA1FR_00/MEDIA1FR/DATA/semantizer_files/"
+            + filename
+            + "_HC.xml",
+            1,
+        )
+        parse_test2(
+            root,
+            channels,
+            filenames,
+            wav_folder,
+            csv_folder,
+            method,
+            task,
+            filename,
+            concepts_full,
+            concepts_relax,
+        )
+
+
+def parse(
+    root,
+    channels,
+    filenames,
+    wav_folder,
+    csv_folder,
+    method,
+    task,
+    corpus,
+):
+    """
+    Parse data for the train, dev and test csv files of the Media dataset.
+    Files are stored in MEDIA1FR_00/MEDIA1FR/DATA/.
+    They are the original xml files used by the community for train, dev and test.
+
+    Arguments
+    ---------
+    root: Document
+        Object representing the content of the Media xml document being processed.
+    channels: list of str
+        Channels (Right / Left) of the stereo recording to keep.
+    filenames: list of str
+        Linked IDs of the recordings, for the channels to keep.
+    wav_folder: str
+        Path where the wavs will be stored.
+    csv_folder: str
+        Path where the csvs will be stored.
+    method: str
+        Either 'full' or 'relax'.
+    task: str
+        Either 'asr' or 'slu'.
+    corpus: str
+        'train', 'dev' or 'test'.
+
+    Returns
+    -------
+    None
+    """
+
+    for dialogue in tqdm(root.getElementsByTagName("dialogue")):
+
+        speaker_name = get_speaker(dialogue)
+        filename = dialogue.getAttribute("id")
+        channel = get_channel(filename, channels, filenames)
+
+        for turn in dialogue.getElementsByTagName("turn"):
+            if turn.getAttribute("speaker") == "spk":
+
+                time_beg = turn.getAttribute("startTime")
+                time_end = turn.getAttribute("endTime")
+
+                sentences = parse_sentences(
+                        turn, time_beg, time_end, method, task
+                )
+
+                append_data(
+                    wav_folder,
+                    channel,
+                    filename,
+                    speaker_name,
+                    sentences,
+                    corpus,
+                )
+
+    return None
+
+
+def parse_test2(
+    root,
+    channels,
+    filenames,
+    wav_folder,
+    csv_folder,
+    method,
+    task,
+    filename,
+    concepts_full,
+    concepts_relax,
+):
+    """
+    Prepares the data for the csv files of the Media dataset.
+    Files are stored in /E0024/MEDIA1FR_00/MEDIA1FR/DATA/semantizer_files/.
+    They are xml files made after the first dataset release, and have never been used before this recipe.
+    These xml files are structured differently from the original ones, explaining special functions for the test2.
+    We made the decision to make a new corpus from them, named "test2".
+    This new corpus can be used as a second inference corpus, as the original test.
```

Comment:
Put it in the beginning. People tend to read only the first lines. This is the reason why the function exist so it needs to be first not a the end